### DISCUSSION TITLE
Look Library in Account

### DISCUSSION THREAD

**Discussion by mirronake** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#112] [Q&A] [answered]
Hello 👋
Where i'm can see my library or purchase history using API ?

**Comment by LeleDerGrasshalmi** [comment, marked as answer]
## Library
GET `https://library-service.live.use1a.on.epicgames.com/library/api/public/items?includeMetadata=true`
> requires the `library:public:items READ` Permission

## Purchase History
GET `https://www.epicgames.com/account/v2/payment/ajaxGetOrderHistory?sortDir=DESC&sortBy=DATE&locale=de`
> this is a web api, so it requires web (cookie) auth, to view another page use the `nextPageToken` Query Param, which is the ISO-Date of the Last element

Technically theres a real Purchase-History API (Which is behind that web api) but not easy for unexperienced people to get a token with that client (fncom)
